apps/prairielearn/src/migrations/20240712152823_lti13_course_instances__add_lineitems.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE lti13_course_instances
+ADD COLUMN IF NOT EXISTS lineitems TEXT;
+
+CREATE TABLE IF NOT EXISTS lti13_lineitems (
+  id BIGSERIAL PRIMARY KEY,
+  assessment_id BIGINT NOT NULL UNIQUE REFERENCES assessments ON DELETE CASCADE ON UPDATE CASCADE,
```

Comment:
Re-raising for discussion from another resolved comment: should we add a constraint forbidding multiple line items for the same assessment? We can discuss at the dev meeting this week if that's best.

Reply:
Doesn't the UNIQUE there do that?

Reply:
Doh, I'm not used to looking out for `UNIQUE` inline like this. You're absolutely right.